README.md

```diff
@@ -147,6 +147,10 @@ render_component(
 
     # An optional object which will be used instead of the default renderer
     renderer=None,
+
+    # An optional dictionary of request header information (such as `Accept-language`)
```

Comment:
This could be a good spot to put in a sample dictionary (like props does above).